Sources/MongoKitten/AggregateBuilder.swift

```diff
@@ -52,6 +52,27 @@ public struct AggregateBuilder {
 }
 
 extension MongoCollection {
+	/// The `aggregate` command will create an `AggregateBuilderPipeline` where data can be aggregated
+	/// and be transformed in multiple `AggregateStage` operations
+	///
+	/// With Swift > 5.1 you can use the function builders instead of the `aggregate(_ stages: [AggregateBuilderStage]) -> AggregateBuilderPipeline` function.
+	///
+	/// # Example:
+	/// ```
+	/// let pipeline = collection.buildAggregate {
+	///    match("name" == "Superman")
+	///    unwind(fieldPath: "$arrayItem")
+	/// }
+	///
+	/// pipeline.decode(SomeDecodableType.self).forEach { yourStruct in
+	///	    // do sth. with your struct
+	///	}.whenFailure { error in
+	///	    // do sth. with the error
+	/// }
+	/// ```
+	///
+	/// - Parameter build: the `AggregateBuilderStage` as function builders
+	/// - Returns: returns an `AggregateBuilderPipeline`
```

Comment:
I'd add something like:

`.. that should be executed to get results`

Reply:
Maybe not perfect, but I'm wondering if people understand that part. That it's not a sent query yet.

Reply:
You're right. I will add another aggregation query like lookup